File: web/src/assets/scripts/Cell.js
/**
 * 定义一个蛇的格子为Cell
 */
export class Cell{
    constructor(r, c){
        this.r = r;
        this.c = c;
        this.x = c + 0.5;//中心点坐标
        this.y = r + 0.5;//中心点坐标
    }
}

File: backend/src/main/java/com/kob/backend/consumer/utils/Cell.java
package com.kob.backend.consumer.utils;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.extern.slf4j.Slf4j;

@Slf4j
@Data
@NoArgsConstructor
@AllArgsConstructor
/**
 * @description 表示组成蛇的每一个单元
 */
public class Cell {
    private Integer x;
    private Integer y;
    public void testLog(){
        int k = 9;
        log.info("a{}",k);
    }
}
